modules/openapi-generator/src/main/java/org/openapitools/codegen/DefaultCodegen.java

```diff
@@ -3626,6 +3627,13 @@ public boolean isDataTypeFile(String dataType) {
                 } else if ("bearer".equals(securityScheme.getScheme())) {
                     cs.isBasicBearer = true;
                     cs.bearerFormat = securityScheme.getBearerFormat();
+                } else if ("signature".equals(securityScheme.getScheme())) {
```

Comment:
One issue here is that the "signature" scheme has not been registered with IANA yet. This value may have to be changed when it is actually registered.